lib/auth0/api/v2/user_blocks.rb

```diff
@@ -0,0 +1,57 @@
+module Auth0
+  module Api
+    module V2
+      # Methods to use the User Blocks endpoints
+      module UserBlocks
+        attr_reader :user_blocks_path
+
+        # Retrieves the user blocks
+        # @see https://auth0.com/docs/api/management/v2#!/User_Blocks/get_user_blocks
+        # @param identifier [string]  Should be any of: username, phone_number, email.
+        #
+        # @return [json] the user blocks
+        def user_blocks(identifier)
+          raise Auth0::InvalidParameter, 'Must specify a valid identifier' if identifier.to_s.empty?
+          path = "#{user_blocks_path}?identifier=#{identifier}"
```

Comment:
looks like the get method does not support query string. Do we want to implement it?